lib/util/tar-create-options.js

```diff
@@ -1,10 +1,16 @@
+const zlib = require('zlib')
 const isPackageBin = require('./is-package-bin.js')
 
 const tarCreateOptions = manifest => ({
   cwd: manifest._resolved,
   prefix: 'package/',
   portable: true,
-  gzip: true,
+  gzip: {
+    // forcing the compression strategy to Z_RLE
+    // yields a bit-identical, though considerably
+    // larger, end result in both arm64 and x86_64.
```

Comment:
Hmm... reproducible builds are good, but I feel like sending considerably more data over the wire is not great.

Reply:
Is it any less bad if you add

```js
{
  windowBits: 15,
  memLevel: 9,
  level: 9,
  strategy: 3,
}
```

Reply:
I'm confused, is the dir fetcher here in pacote also what libnpmpack uses to create the tarball for upload?

Reply:
those settings make no difference at all.

also, confirmed that yes this is indeed the module that packs a tarball for publishing. so, yes, you're right, this is not what we want to do by default. i can tweak this so that this setting will only take place as part of fetching and packing a remote git repository.